core/lib/server/pubtype.ts

```diff
@@ -92,12 +92,22 @@ export const getAllPubTypesForCommunity = (communitySlug: string) => {
 						.selectFrom("_PubFieldToPubType")
 						.whereRef("B", "=", "pub_types.id")
 						.select((eb) =>
-							eb.fn.coalesce(eb.fn.agg("array_agg", ["A"]), sql`'{}'`).as("fields")
+							eb.fn
```

Comment:
instead of returning a list of field Ids `PubFieldsId[]`, we now return a list of `{id: PubFieldsId, isTitle: boolean}`